compiler/src/iree/compiler/Dialect/HAL/Transforms/OutlineMemoizeRegions.cpp

```diff
@@ -153,6 +156,8 @@ static IREE::Util::FuncOp outlineMemoizeRegionBody(
                                                          name, funcType);
   moduleSymbolTable.insert(funcOp);
   funcOp.setVisibility(SymbolTable::Visibility::Private);
+  // funcOp.setInliningPolicyAttr(
+  // moduleBuilder.getAttr<IREE::Util::InlineNeverAttr>());
```

Comment:
intentionally commented out?

Reply:
yeah was just testing - fixed (was taking an hour or two to get the model compiled :)